pkg/shell/shell.css

```diff
@@ -627,6 +627,16 @@ ul.available-interfaces-group {
     vertical-align: top;
 }
 
+/* workaround: make text red for select if contained in has-error, not in patternfly */
+.has-error > .bootstrap-select > button > span {
+    color: #A94442;
+}
+
+
+.containers-run-message {
```

Comment:
I don't think so. patternfly.css contains this CSS, which is a polyfill for exactly that reason:

```
[hidden],
template {
  display: none;
}
```

The polyfill works all the way back to IE6. So we should be using <el hidden> as a way to start elements in a 'display: none' state. We should then generally be using the jQuery ```.show()``` ```.hide()``` and ```.toggle()``` to manipulate that state.

Obviously there are exceptions where an entire set of objects are hidden if their parent has a certain CSS class. It's fine to write custom CSS for those. But for one off cases, starting an element as hidden, we should just use the hidden attribute.